miio/vacuum.py

```diff
@@ -412,19 +471,50 @@ def disable_dnd(self):
         return self.send("close_dnd_timer", [""])
 
     @command(
-        click.argument("speed", type=int),
+        click.argument("speed", type=str),
     )
-    def set_fan_speed(self, speed: int):
+    def set_fan_speed(self, speed: [int, FanSpeed]):
```

Comment:
This could be an `Union`, but I think it is better to keep it as integer; or is there some specific need why both should be accepted?